api/v1alpha1/workspace_validation.go

```diff
@@ -78,7 +78,7 @@ func (w *Workspace) validateCreate() (errs *apis.FieldError) {
 	return errs
 }
 
-func (w *Workspace) validateUpdate(old *Workspace) (errs *apis.FieldError) {
+func (w *Workspace) validateUpdateWorkspace(old *Workspace) (errs *apis.FieldError) {
```

Comment:
You don't need to rename it because this method is associated with the object type workspace. Adding workspace in the suffix does not produce any extra useful information.